services/core-web/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js

```diff
@@ -108,15 +118,15 @@ export class NoticeOfWorkTable extends Component {
             value={selectedKeys[0] || this.props.searchParams[field]}
             onChange={(e) => setSelectedKeys(e.target.value ? [e.target.value] : [])}
```

Comment:
[field] is different than the other patterns, is this due to the filter within the table?

Reply:
Yup!